project1/Schema.py

```diff
@@ -0,0 +1,17 @@
+from flask_sqlalchemy import SQLAlchemy 
+from datetime import datetime
+from passlib.hash import sha256_crypt
+db = SQLAlchemy()
+
+class schema(db.Model):
```

Comment:
From the next time please make the class name according to the purpose. Like for example in this problem, since we are adding users to the database, we can name class name as users. schema is a generic word.